src/Rails.elm

```diff
@@ -252,15 +249,23 @@ request { method, headers, url, body, expect, timeout, tracker } =
         accept =
             case expect of
                 ExpectJson _ ->
+                    -- q indicates our preference, from 0 to 1. We almost never
+                    -- want a fallback, so we indicate that.
```

Comment:
🎨 maybe clarify this comment to say expicitly that "*/*; q=0.01" indicates that we prefer this option the least.